RecoTracker/LSTCore/src/alpaka/LST.cc

```diff
@@ -26,26 +26,26 @@ namespace {
     std::vector<unsigned int> hits;
 
     unsigned int maxNHits = 0;
-    if (trackCandidateType == 7)
-      maxNHits = Params_pT5::kHits;  // pT5
-    else if (trackCandidateType == 5)
-      maxNHits = Params_pT3::kHits;  // pT3
-    else if (trackCandidateType == 4)
-      maxNHits = Params_T5::kHits;  // T5
-    else if (trackCandidateType == 8)
-      maxNHits = Params_pLS::kHits;  // pLS
+    if (trackCandidateType == lst::LSTObjType::pT5)
```

Comment:
Very minor thing, but here and elsewhere I don't think you need the `lst::` (except in the standalone files)

Reply:
I will change it.